src/main/scripts/ncm-ncd

```diff
@@ -183,6 +155,41 @@ sub app_options()
             HELP    => "Chroot to the the directory given as an argument",
             DEFAULT => undef
         },
+
+        # Advanced options
+        {
+            NAME => 'ignorelock',
+            HELP => 'ignore application lock. Use with care.'
+        },
+
+        {
+            NAME => 'forcelock',
+            HELP => 'take over application lock. Use with care.'
+        },
+
+        {
+            NAME    => 'nodeps',
+            HELP    => 'ignore broken (pre/post) dependencies in configure. Use with care.',
+            DEFAULT => undef
+        },
+
+        {
+            NAME    => 'nodepsnoerrors',
+            HELP    => 'errors from broken (pre/post) dependencies in configure are downgraded to warnings (implies --nodeps --autodeps). Use with care.',
```

Comment:
What is the meaning of "broken" in this context? Running to completion but with ERRORs or some sort of code exception? (i.e. code is buggy)